config/settings/base.py

```diff
@@ -389,10 +389,9 @@
 # --------------------------------------------------------------------------
 ACCOUNT_ADAPTER = "adserver.auth.adapters.AdServerAccountAdapter"
 ACCOUNT_USER_MODEL_USERNAME_FIELD = None
-ACCOUNT_EMAIL_REQUIRED = True
-ACCOUNT_USERNAME_REQUIRED = False
 ACCOUNT_LOGIN_ON_PASSWORD_RESET = True
 ACCOUNT_MAX_EMAIL_ADDRESSES = 1
+ACCOUNT_SIGNUP_FIELDS = ["email*", "password1*", "password2*"]
```

Comment:
The * here seems a bit magical -- worth a comment, or a link to the docs?

Reply:
Added